migrations/20210412140907-remove-notifications-unique-type.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      DROP INDEX IF EXISTS "Notifications_type_CollectiveId_UserId";
+    `);
+
+    await queryInterface.addIndex('Notifications', ['CollectiveId', 'type', 'channel']);
```

Comment:
This index will be more performant: `CollectiveId` is that main way to filter notifications